pyiceberg/catalog/rest.py

```diff
@@ -719,12 +721,15 @@ def _remove_catalog_name_from_table_request_identifier(self, table_request: Comm
             )
         return table_request
 
-    @retry(**_RETRY_ARGS)
```

Comment:
Did we remove the retry decorator intentionally?

Reply:
No, I did this earlier when I was trying another route. Thanks for catching this :)